f2/src/AnonymousPage.js

```diff
@@ -27,14 +28,14 @@ export const AnonymousPage = () => {
               <Lead>
                 <Trans id="anonymousPage.intro" />
               </Lead>
-
+              {console.log(doneForms)}
```

Comment:
Was this added for testing? should we remove it?

Reply:
Thought i removed it! Can we just remove it before merging?

Reply:
whoa, good catch @khalidelaggan ! Removed.

Reply:
was it removed from master as well?

Reply:
Nope, just this PR